lib/oppen/print_stack.rb

```diff
@@ -92,7 +92,7 @@ def print(token, token_width, trim_on_break: 0)
 
     # Handle Begin Token.
     #
-    # @param token [Token]
+    # @param token       [Token]
     # @param token_width [Integer]
```

Comment:
This is an internal `@param`, so no need to explain it, and it's really mega obvious, so it's fine to leave it empty.

We just need to be more strict on public APIs.